frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -183,7 +140,12 @@ const HostSummary = ({
         {!deviceUser && isPremiumTier && renderHostTeam()}
         <div className="info-flex__item info-flex__item--title">
           <span className="info-flex__header">Disk space</span>
-          {renderDiskSpace()}
+          {renderDiskSpaceGraph(
```

Comment:
Similar to the last comment about DiskSpaceCleanup. Because we are exporting it as a function we have a less intuitive function call here instead of the JSX DiskSpaceComponent.